packages/itwin/grouping-mapping-widget/src/widget/components/GroupAction.tsx

```diff
@@ -281,64 +345,53 @@ const GroupAction = ({
         </Fieldset>
         <Fieldset legend='Group By' className='gmw-query-builder-container'>
           <RadioTileGroup
-            className="gmw-radio-group-tile"
+            className='gmw-radio-group-tile'
```

Comment:
Inconsistent. I see that in most cases double quotes are used, so please stick to double quotes.

Reply:
I remember either eslint or prettier use single quote for component attribute; double quote for rest of strings. @Guillar1 Another formatting problem

Reply:
Do we use prettier?